spec/features/friendrequest2_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.feature 'Friendrequest2s', type: :feature do
+  let(:user1) { FactoryBot.create(:user) }
+  before do
+    user1.confirm
+  end
+
+  let(:user2) do
+    User.create(first_name: 'name', last_name: 'surename',
+                email: 'anyemail@gmail.com', password: 'password',
+                password_confirmation: 'password')
+  end
```

Comment:
Please read that article
https://www.driftingruby.com/episodes/sample-data-with-factory-bot-and-faker

Then, learn how to use FactoryBot with Faker to simplify the lines above